product_docs/docs/pgd/5/logical-slot-failover.mdx

```diff
@@ -0,0 +1,94 @@
+
+Currently Logical Slot Failover support is a global option that is controlled by a top-group option. The failover_slot_scope top-group option can currently be set to (and defaults to) local which disables replication of logical slots or `global` which enables the replication of all non-temporary logical slots created in the PGD database. 
+
+Temporary logical slots will not be replicated as they have a lifetime scoped to the session that created them and will go away when that session ends.
+
+### At-least-once delivery guarantees
+
+Logical Slot Failover support takes steps to ensure that the consumer receives all changes at least once. This is done by holding back slots until delivery has been confirmed, at which point the slot is then advanced on all nodes in an asynchronous manner. In the case of a failure on the node where the slot was being consumed, the slot would be held until the consumer connected to a node in the cluster. This would then allow the slot to progress.
+
+!!! Important  
+If a consuming application disconnects and doesn’t reconnect the slot will remain on held back on every node in the cluster. As this consumes disk and memory, it is essential that this situation is avoided; applications which consume slots must return to consuming as soon as possible.  
+!!!  
+
+### Exactly-once delivery
+
+Currently, there is no way to ensure exactly-once delivery, and we expect consuming applications to manage the discarding of previously completed transactions. 
+
+## Enabling Logical Slot Failover Support
+
+To enable Logical Slot Failover support run the SQL command and call the [`bdr.alter_node_group_option`](/pgd/5/reference/nodes-management-interfaces#bdralter_node_group_option) function with the following parameters:
+
+```sql
+select bdr.alter_node_group_option(<top-level group name>,
+     'failover_slot_scope',
+     'global');
+
+```
+
+Replacing `<top-level group name>` with the name of your cluster’s top level group.
+
+If you do not know the name, it is the group with a node_group_parent_id equal to 0 in `[`bdr.node_group`](/pgd/5/reference/catalogs-visible#bdrnode_group)`. You can also use:
+
+```sql
+SELECT bdr.alter_node_group_option(
+     node_group_name,
+     'failover_slot_scope',
+     'global')
+    from bdr.node_group	
+    where node_group_parent_id=0;
+```
+
+To ensure you are setting the correct, top-level group’s option.
+
+Once enabled you can use:
+
+```sql
+SELECT pg_create_logical_replication_slot('myslot',
+                                          'test_decoding');
+```
+
+To create a new globally replicated slot.
+
+Note that logical replication slots created before the option is set to `global` will not be replicated. Only new slots will be replicated.
```

Comment:
`bdr.failover_replication_slots` catalog stores the information about all the Logical Failover Slots. User can query that catalog to check if a slot is created with failover support or not.
```
CREATE TABLE bdr.failover_replication_slots
(
    slot_name name NOT NULL PRIMARY KEY,
    slot_id oid NOT NULL,
    node_group_id oid NOT NULL,
    plugin name NOT NULL,
    twophase boolean NOT NULL,
    active_node oid,
    active_pid int,
    prev_node oid,
    UNIQUE(slot_id)
) WITH (user_catalog_table=true);
```